src/main/java/eu/execom/pomodoroTeam/entities/UserEntity.java

```diff
@@ -41,5 +43,7 @@
                     @JoinColumn(name = "Team_id", nullable = false, updatable = false)})
     private List<TeamEntity> teams = new ArrayList<>();
 
+	
+
   
 }

```

Comment:
Please remove empty space. Use at most one empty line to separate different blocks of code.